docs/reference/contributing/guidelines/workflow.md

```diff
@@ -1,42 +1,60 @@
+Responsibilities:
 
-Patch contributions can only be accepted through GitHub by creating a pull request from forked versions of our repositories. This allows us to review the contributions in a user friendly and reliable way, under public scrutiny.
+1. Check for CLA (Contributor License Agreement) compliance.
+1. Ensure the relevant stakeholders review pull requests.
+1. Guide contributors both technically and procedurally.
+1. Run pull requests through the CI systems.
+1. Put label version.
+1. Merge pull requests into the requested branches.
+1. Make periodic patch and feature releases.
 
-Please create separate pull requests for each concern; each pull request should have a clear unity of purpose. In particular, separate code formatting and style changes from functional changes. This makes each patch’s true contribution clearer and therefore quicker and easier to review.
+The current maintainers are:
 
-#### Reporting and fixing bugs
+* Anna Bridge [adbridge](https://os.mbed.com/users/AnnaBridge).
+* Martin Kojtal [0xc0170](https://os.mbed.com/users/Kojto).
+* Jimmy Brisson [theotherjimmy](https://os.mbed.com/users/theotherjimmy).
+* Shrikant Tudavekar [studavekar](https://os.mbed.com/users/shrikant1213).
+* Sam Grove [sg-](https://os.mbed.com/users/sam_grove).
+* Cruz Monrreal [cmonr](https://os.mbed.com/users/MrCruz).
 
-Before submitting a bug report or a bug fix, please [discuss it on the forums](https://os.mbed.com/forum/bugs-suggestions/) to avoid duplication of work, as we or others might be working on it already.
+##### Stakeholders
 
-##### Bug reports (issues) on GitHub
+They are responsible for a specific mbed OS module or feature.
 
-All Mbed OS is on GitHub; please use GitHub's [issues mechanism](https://guides.github.com/features/issues/) to open a bug report directly against the relevant GitHub repository.
+Responsibilities:
 
-##### Bug fixes
+1. Review pull request and coordinate reviews within the team
+1. Ensure the version label is correct
 
-Please refer to the [code contributions chapter](/docs/v5.7/reference/guidelines.html#style).
+#### Contributions
 
-A member of the Mbed team must verify bug fixes before we pull the fixes into the main branch. You must therefore use GitHub to fork the repo and then submit a pull request with your changes.
+All code changes and additions to Mbed OS are handled through GitHub. If you want to contribute, either by adding features or by fixing bugs, please follow the guidelines for [new features](#contributing-new-features-to-mbed-os) and [bugs](#reporting-and-fixing-bugs). In both cases, please follow the [code style guide and GitHub pull request guidelines](/docs/v5.7/reference/guidelines.html#style)</a>. Please also read the [CLA](/docs/v5.7/reference/guidelines.html#cla) guidelines because we will immediately close pull requests submitted without a CLA.
```

Comment:
Please fix the first two links because they're now broken.

Reply:
I did not change those links, only moved this paragraph. Any help to make it right if they became invalid? don't know how to yet